.travis.yml

```diff
@@ -0,0 +1,29 @@
+# Travis configuration file using the build matrix feature
+# Read more under http://docs.travis-ci.com/user/build-configuration/
+# THIS SCRIPT IS SUPPOSED TO BE AN EXAMPLE. MODIFY IT ACCORDING TO YOUR NEEDS!
+
+sudo: false
+language: python
+virtualenv:
+  system_site_packages: true
+env:
+  matrix:
+    - DISTRIB="conda" PYTHON_VERSION="3.3" COVERAGE="true"
+    - DISTRIB="conda" PYTHON_VERSION="3.4" COVERAGE="false"
+addons:
+  apt:
+    packages:
+      - git
+      - python-pip
+install:
+  - source tests/travis_install.sh
```

Comment:
Hi @sammacbeth.  Thanks for the comments.  The advantage of using conda is that it speeds up the testing process *considerably*.  When using the Travis-provided python, you have to pip install each package, which, particularly for scipy and numpy takes a long time, as these are then built from source.  Conda solves all of this in a neat way.